scalableminds/webknossos
# task assignment for admins

- also fixes the project readAccesQ 
 
### Mailable description of changes:
 - admins can now get tasks from all projects in their organization

### Steps to test:
- create a project and task for a team other than the admin's team
- try to get a task as the admin
- you should be able to get a task from that project

### Issues:
- fixes #2684 

------
